docs/guides/saturn-0/whats-new.mdx

```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
 
 ::: tip NOTE
 Saturn 0 has been successfully voted in by the Protocol DAO and Oracle DAO and is scheduled to go live on `28 October 2024, 00:00 UTC`.
-This upgrade consists parameter changes, [Houston Hotfix (RPIP-63)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x1d75c5247cf457d26fbf7f26622c21de912565256fc3273b92a40aa14dd1af3d)
-and [Tokenomics Rework Prelude (RPIP-62)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x7f6c978251e0ae752c619cc459331e5b3cc081aed13b133159b43fb6194b3304).
+This upgrade consists of the changes detailed in [Tokenomics Rework Prelude (RPIP-62)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x7f6c978251e0ae752c619cc459331e5b3cc081aed13b133159b43fb6194b3304).
 :::
 
 
 ## ETH only minipools in Saturn 0 
 
-- The RPL requirement for new minipools are a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed. 
-Node operators can now create minipools with an upfront cost of 8 ETH. In addition, a node operator may stake RPL to qualify for additional rewards 
-and access voting power within the [Protocol DAO](guides/houston/pdao#the-protocol-dao-pdao). 
+- The RPL requirement for new minipools is a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed.
+Node operators can now create minipools with an upfront cost of 8 ETH, which will feature a commission ranging from 10% to 14%. In addition, a node
```

Comment:
```suggestion
Node operators can now create minipools with an upfront cost of 8 ETH, which will feature a commission of 10%. In addition, a node
```
Something like that. If the next sentence talks about additional rewards with RPL separately, this one should probably be just about ETH-only.